### DISCUSSION TITLE
I bought Pro license but any e-mails weren't arrived plz help

### DISCUSSION THREAD

**Discussion by kjk4012** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#890] [Q&A]
I purchased a pro license, but no e-mails were received.

I have solid data such as a purchase receipt, but there is no contact information on the official website, so I leave a message.

plz help

**Comment by MatthiasKerbl** [comment]
Hi @kjk4012,

I am sorry you are having trouble with the buying process.
Please take a look at the following steps from our FAQ:

**I Did Not Get The License Code Or Any Receipt Emails After Buying AlDente Pro!**

There are multiple reasons why you did not get your license code after the purchase:

1. Sometimes it takes up to 30 Minutes for the purchase emails to go through.
2. The purchase emails end up in the junk email folder.
3. You had a typo in the email address during your purchase.
4. Failures of the payment networks/providers.

Please contact the support team of our payment provider Paddle at [assist@paddle.com](mailto:assist@paddle.com) and include the email address you used to purchase AlDente Pro. The Paddle support will try to solve your issue as fast as possible.

Cheers,
Matthias
